pytest_dbfixtures/conf/dbfixtures.conf

```diff
@@ -6,7 +6,7 @@ redis:
     db: 0
 
 mongo:
-    mongo_exec: /usr/bin/mongod
+    mongo_exec: mongod
```

Comment:
change also redis-server and other executors if possible. Add inline comment in config otherwise